README.md

```diff
@@ -1,48 +1,97 @@
+  <a href="#how-it-works">How It Works</a> •
+  <a href="#project-structure">Project Structure</a> •
+  <a href="#learn-more">Learn more</a> •
+  <a href="#learn-more">About us</a>
+</p>
 
-- Install Yarn package manager
+___
 
-### Python
+This project aims to offer a cheaper and more accessible way to perform sleep studies from home. To achieve this, a machine learning classifier is used to automate the manual sleep scoring step. This drastically cuts the time needed to process each and every sleep sequences and it completely eliminates the learning curve associated with manual sleep scoring.
 
-- Install Python 3 and pip
-- Consider using `venv` to create a virtual environment
+🌐 Our web application does exactly all that and is available at [this link](https://polycortex.github.io/polydodo/). Check it out!
 
-### Flutter
+🤖 Our Android app is underway. Give us a star to stay tuned for upcoming news about its release!
 
-- Install the latest stable version of flutter
+**This application is not intended for medical purposes and the data it produces should not be used in such context. Its only goal is to help you study your sleep by yourself. Always seek the advice of a physician on any questions regarding a medical condition.**
 
-### VS Code
+## Key features
 
-- Install VS Code
-- Install the project's recommended extensions
+- Compatible with both OpenBCI's Cyton and Ganglion boards.
+- Automatic sleep stage scoring based on the AASM's labels.
+- A comprehensive guide on how to record polysomnographic EEG data from home.
+- A nice and straightforward UI to help you upload, visualize and save your sleep.
+
+## How it works
+
+Polydodo is composed of two client apps, a web one and a mobile one, through which the user can interact. These clients are not complementary but are alternatives to one another. Each of these clients uses the same local server which hosts the automatic sleep stages classification algorithm.
+
+The web client allows the user to upload a data file acquired using an OpenBCI board and then presents him a detailed and personalized analysis of his sleep. Additionally, this client will further detail the process by which we come to classify sleep in stages and offer a review of this process. OpenBCI boards must be configured via OpenBCI GUI and data must be saved on a SD card (Cyton only) or through a session file.
+
+On the other hand, the mobile client offers a tool that can be used on a regular basis. Unlike the web application, this app can save sleep sequences for later consultation and display the aggregated results of several nights of sleep on a dashboard. Also, it will guide the user from the installation of the electrodes, until the end of his acquisition.
+
+Finally, both these clients use a local http server that is easy to installed. This server is local so that your data is not sent over the internet. Biosignals data are sensitive and this is our way to promise you security.
```

Comment:
```suggestion
Finally, both these clients use a local HTTP server that is easy to install. This server is hosted locally so that your data is not sent over the internet. Biosignals data are sensitive and this is our way to promise you security.
```